api/agent.go

```diff
@@ -333,7 +347,7 @@ func getServerVersion(serverURL string) (string, error) {
 
 // NewControllerWrapper returns a constructor function used to create
 // a new UserController.
-func NewControllerWrapper(config *loadtest.Config, controllerConfig interface{}, userOffset int, namePrefix string, metrics *performance.Metrics) (loadtest.NewController, error) {
+func NewControllerWrapper(config *loadtest.Config, controllerConfig interface{}, userOffset int, namePrefix string, metrics *performance.Metrics, isBrowserAgentInstance bool) (loadtest.NewController, error) {
 	maxHTTPconns := loadtest.MaxHTTPConns(config.UsersConfiguration.MaxActiveUsers)
```

Comment:
We should change this to `MaxActiveBrowserUsers` if it's a browser agent instance:
```suggestion
	maxHTTPconns := loadtest.MaxHTTPConns(config.UsersConfiguration.MaxActiveUsers)
	if isBrowserAgentInstance {
		maxHTTPconns = loadtest.MaxHTTPConns(config.UsersConfiguration.MaxBrowserActiveUsers)
	}
```